lib/base-controller.js

```diff
@@ -3,10 +3,12 @@
 var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
+var mixins = require('./middleware/mixins');
 
 var BaseController = function BaseController(options) {
   this.confirmStep = options.confirmStep || '/confirm';
   Controller.apply(this, arguments);
+  this.use(mixins);
```

Comment:
sounds good! Didn't realise there was a render method that is only called on GET requests - the other methods I extended are called on all requests, this is a better place for it - will refactor